source/Sashimi.Terraform/Validation/TerraformValidator.cs

```diff
@@ -10,54 +10,88 @@
 using Sashimi.Terraform.ActionHandler;
 using Sashimi.Terraform.CloudTemplates;
 using Sprache;
-using PropertiesDictionary = System.Collections.Generic.IReadOnlyDictionary<string, string>;  
 
 namespace Sashimi.Terraform.Validation
 {
-    public class TerraformValidation : IDeploymentActionValidator
+    public class TerraformValidator : AbstractValidator<DeploymentActionValidationContext>
```

Comment:
This file didn't actually change much, moving `AddAzureAccountRules()` and `AddAwsAccountRules()` complicated the diff. Is there a reason you moved them up?

Reply:
not sure I understand, those methods are part of the `TerraformValidator` hence the reason to move them

Reply:
See how [AddAzureAccountRules](https://github.com/OctopusDeploy/Sashimi/pull/10/files/416f6be835f7a0fb04c2f799ed3b06bd658aa171#diff-e6a0c8f9a78f2d263d0b859d775ee6edL132) was on Line 132, now it's moved up to Line 48 etc.

The functions were moved up in the file

Reply:
oops, didn't even realise it